src/Symfony/Bridge/Twig/Tests/Extension/FormExtensionBootstrap3HorizontalLayoutTest.php

```diff
@@ -119,35 +119,42 @@ protected function setTheme(FormView $view, array $themes)
     public function testRange()
     {
         // No-op for forward compatibility with AbstractLayoutTest 2.8
```

Comment:
I would move this comment into the message of the incomplete test instead, so that running in verbose mode explains what is incomplete